corehq/apps/api/odata/tests/test_feed.py

```diff
@@ -21,6 +22,7 @@
 from corehq.util.elastic import reset_es_index
 
 
+@attr(es_test=True)
```

Comment:
One thing I was hesitant to suggest before, but is maybe easier now that this is split out into a separate PR, is to make a custom decorator rather than using `attr` in many places. It reduces duplication a tiny bit, but more importantly makes it easier to detect a typo (which would cause an error instead of passing silently while doing the wrong thing). For example imagine someone was attempting to tag another ES test, but accidentally typed this:
```py
@attr(ex_test=True)
```
I suggest we define a custom decorator for ES tests somewhere like _corehq/apps/es/tests/utils.py_:
```py
def es_test(test):
    """Decorator for tagging ElasticSearch tests

    :param test: A test class, method, or function.
    """
    return attr(es_test=True)(test)

Reply:
Sure, Daniel